.evergreen/run-tests.sh

```diff
@@ -139,6 +139,25 @@ if test -n "$FLE"; then
 
   export LIBMONGOCRYPT_PATH=`pwd`/rhel-70-64-bit/nocrypto/lib64/libmongocrypt.so
   test -f "$LIBMONGOCRYPT_PATH"
+
+  echo "Waiting for mock KMS servers to start..."
+   wait_for_kms_server() {
+      for i in $(seq 60); do
+         # Exit code 7: "Failed to connect to host".
+         if curl -s "localhost:$1"; test $? -ne 7; then
```

Comment:
Wouldn't it better to require status 0 than anything other than 7?

Reply:
Some of mock servers we start have bad configurations - wrong certificates, etc. So, curl won't be able to connect to them "good enough" to return 0. Hence check for "anything but Failed to connect".

Reply:
How about adding the `-k` option in this case?

Reply:
Just adding `-k` didnt't work. I end up with this - `curl -d '' -X POST -k https://localhost:8002/`, which works (== returns 0) for almost all mock servers. Unfortunately, an attempt to connect to mock server with `--require_client_cert` option will give us error code 56.

Reply:
OK, whatever you think is best is good with me.